pkg/database/migrations.go

```diff
@@ -893,6 +893,26 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 				return nil
 			},
 		},
+		{
```

Comment:
Why not `gorm.AutoMigrate`? Because you can't set a NOT NULL constraint and DEFAULT at the same time when there are existing rows in the database. Postgres creates them with NULL, then backfills the default... but that violates the NOT NULL constraint. So we have to create the column as nullable, wait for the tx to finish, then mark it as not nullable.